homeassistant/components/ozw/light.py

```diff
@@ -91,9 +130,6 @@ def async_set_duration(self, **kwargs):
         128-254 = 1 minute to 127 minutes
         255     = factory default
         """
-        if self.values.dimming_duration is None:
```

Comment:
Is it safe to remove this check?

Reply:
Think I nuked this in error, putting it back.